### PR TITLE
Update help for irrelevantGroups

### PR DESCRIPTION
**Pull request by jtnord**
@reviewbybees

### CODE REVIEW

**Comment by andresrc** [comment]
:bee:

File: src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-removeIrrelevantGroups.html [hunk 1]
```diff
@@ -3,4 +3,5 @@
     This can significantly improve performance in environments with a large number of groups
     but a small number of corresponding rules defined by the Authorization Strategy.
     The user is required to re-login in order for the changes to take effect.
+    <br><strong>Note:</strong> this setting may be incompatable with some Authorzation Strategies.
```
Comment (recena) [inline src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-removeIrrelevantGroups.html]: I recommend you to use:

```
<p>
  <strong>Note:</strong> this setting may be incompatable with some Authorzation Strategies.
</p>
```

**Comment by recena** [comment]
:bee:

**Comment by jtnord** [comment]
@reviewbtbees done :-)

File: src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-removeIrrelevantGroups.html (hunk 1 above)
Comment (medianick) [inline src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-removeIrrelevantGroups.html]: @recena, you've got a couple typos in your recommended version, corrected below:

    <p>
      <strong>Note:</strong> this setting may be incompatible with some Authorization Strategies.
    </p>

**Comment by nnordrum** [comment]
Any clarity on what Strategies are known to be affected, what might cause a Strategy to be affected, and/or examples of what the effect might look like? This is very doom and gloom with nothing to provide any reassurance that everything isn't going to go to hell in a weird unpredictable way.

**Pull request merged**